autopush/web/validation.py

```diff
@@ -35,6 +35,9 @@
 AUTH_SCHEMES = ["bearer", "webpush"]
 PREF_SCHEME = "webpush"
 
+# Base64 URL validation
+VALID_BASE64_URL = re.compile(r'^[0-9A-Za-z\-_=]*$')
```

Comment:
is `foo=bar` a valid topic name? maybe `r'^[a-zA-Z0-9\-_]+=*$'` to more closely match Base64?